### ISSUE TITLE
"fatal error: windows.h: No such file or directory" on Linux

### ISSUE THREAD

**Issue by JayFoxRox** [MADEAPPS/newton-dynamics#161]
```
newton-dynamics/applications/demosSandbox/sdkDemos/toolBox/FileBrowser.cpp:16:10: fatal error: windows.h: No such file or directory
```

After doing:

```
git clone https://github.com/MADEAPPS/newton-dynamics.git
cd newton-dynamics
mkdir build
cd build
cmake ..
make
```

x64 Linux, GCC 8.2.1, newton-dynamics 83cff98

**Comment by JulioJerez** [comment]
fixed, please try again

**Comment by JayFoxRox** [comment]
Fixed in 913197fa797f61603f165847c6146a77d5444a89

*There's a lot of other crashes with the sandbox, but I'll create individual issues when I find the time (if I end up using Newton in my software - which is yet to be decided).*
